src/Stratis.Bitcoin/Configuration/Logging/LoggingConfiguration.cs

```diff
@@ -29,36 +28,42 @@ public class ExtendedLoggerFactory : LoggerFactory
 
         public static ILoggerFactory Create()
         {
+            return new CustomLoggerFactory();
+            /*
             return Create(builder =>
             {
                 builder
-                    .AddFilter("Default", LogLevel.Information)
-                    .AddFilter("System", LogLevel.Warning)
-                    .AddFilter("Microsoft", LogLevel.Warning)
-                    .AddFilter("Microsoft.AspNetCore", LogLevel.Error);
+                    .AddFilter("Default", Microsoft.Extensions.Logging.LogLevel.Information)
+                    .AddFilter("System", Microsoft.Extensions.Logging.LogLevel.Warning)
+                    .AddFilter("Microsoft", Microsoft.Extensions.Logging.LogLevel.Warning)
+                    .AddFilter("Microsoft.AspNetCore", Microsoft.Extensions.Logging.LogLevel.Error);
             }
             );
+            */
         }
 
         /// <summary>Loads the NLog.config file from the <see cref="DataFolder"/>, if it exists.</summary>
         public static ILoggerFactory Create(LogSettings settings, DataFolder dataFolder)
         {
+            return new CustomLoggerFactory();
+            /*
```

Comment:
This code is commented out, should this be removed?

Reply:
See above.

Reply:
Resolved.